src/peft/tuners/lora/layer.py

```diff
@@ -116,6 +155,21 @@ def __init__(self, base_layer: nn.Module, ephemeral_gpu_offload: bool = False, *
         self.in_features = in_features
         self.out_features = out_features
 
+    def resolve_lora_variant(self, *, use_dora: bool, **kwargs) -> Optional[LoraVariant]:
+        """
+        Return any possible LoRA variant for this layer type.
```

Comment:
```suggestion
        Return a matching LoRA variant for this layer type.
```

To me this suggesated multiple and/or arbitrary LoRA variants.